index.d.ts

```diff
@@ -290,6 +290,12 @@ export declare interface TracerOptions {
    */
   reportHostname?: boolean
 
+  /**
+   * A string representing the minimum tracer log level to use when debug logging is enabled
+   * @default 'debug'
+   */
+  logLevel?: 'error' | 'debug'  
```

Comment:
This should be tested in `docs/test.ts`.

Reply:
Updated (I think? I added logLevel to the init block in the test.ts section)